client/components/credit-card-form-fields/index.jsx

```diff
@@ -98,28 +98,19 @@ export class CreditCardFormFields extends React.Component {
 	};
 
 	handleFieldChange = event => {
-		const { name: fieldName, value: nextValue, options, selectedIndex } = event.target;
-
-		const newState = {};
-
-		if ( fieldName === 'country' ) {
-			newState.countryCode = nextValue;
-			newState.countryName = options[ selectedIndex ].text;
-			if ( ! this.state.phoneCountryCode ) {
-				newState.phoneCountryCode = nextValue;
-			}
-		}
-
-		this.setState( newState, () => this.updateFieldValues( fieldName, nextValue ) );
+		this.updateFieldValues( event.target.name, event.target.value );
 	};
 
 	shouldRenderEbanx() {
-		return isEbanxEnabledForCountry( this.state.countryCode );
+		return isEbanxEnabledForCountry( this.getFieldValue( 'country' ) );
 	}
 
 	renderEbanxFields() {
 		const { translate, countriesList } = this.props;
-		const { countryCode, phoneCountryCode, countryName } = this.state;
+		const { userSelectedPhoneCountryCode } = this.state;
+		const countryCode = this.getFieldValue( 'country' );
+		const countryData = find( countriesList.get(), { code: countryCode } );
+		const countryName = countryData ? countryData.name : translate( 'this country' );
```

Comment:
:disappointed: 76 existing translations will be lost with this change.
If the string must change, the following option has already been translated **36** times:
`translate( 'Country', { context: 'Domain Edit Contact Info form.'} )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/sync-billing-country-across-payment-methods)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
I am not sure what the i18n bot is saying here (it's a new string, not a changed string); however, I am also refactoring the code a bit to improve it so this extra string won't wind up being necessary anyway.